src/test/java/hw2/ex2/DifferentElementsPageUITest.java

```diff
@@ -0,0 +1,66 @@
+package hw2.ex2;
+
+import hw2.AbstractBaseTest;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DifferentElementsPageUITest extends AbstractBaseTest{
+
+    @Test
+    public void checkDifferentElementsPageUIElements() {
+
+        // 2. Assert Browser title
+        assertTrue(waitForTitle("Home Page"));
+
+        // 3. Perform login
+        waitForElementLocatedBy(By.id("user-icon")).click();
+        waitForElementLocatedBy(By.id("name")).sendKeys("Roman");
+        waitForElementLocatedBy(By.id("password")).sendKeys("Jdi1234");
+        waitForElementLocatedBy(By.id("login-button")).click();
+
+        // 4. Assert User name in the right-top side of screen that user is loggined
+        assertEquals(waitForElementLocatedBy(By.id("user-name")).getText(), "ROMAN IOVLEV");
+
+        // 5. Open through the header menu Service -> Different Elements Page
+        waitForElementLocatedBy(By.xpath("//header//*[contains(text(), 'Service')]")).click();
+        waitForElementLocatedBy(By.xpath("//header//*[contains(text(), 'Different elements')]")).click();
```

Comment:
These locators could be simplified. Please look carefully to the elements on the page?